_docs/install/prerequisites/devicepresentation.md

```diff
@@ -7,51 +7,14 @@ module: install/prerequisites/devicepresentaion.md
 
 # Device presentation
 
-StorageOS requires one of two runtime requirements to present virtual block devices
-from userspace. While not mandatory, the NBD aproach does offer significantly
-better performance over the loop-device alternative.
+StorageOS is developed using Linux-IO (LIO) Target. An open-source implementation of the SCSI target. StorageOS uses a 
+software interface for Unix-like computer operating systems that lets non-privileged users create their own file systems without editing kernel code.
```

Comment:
Remove the line "StorageOS uses a.... kernel code".

Reply:
Thank you! Done https://github.com/storageos/storageos.github.io/pull/148/commits/85f8b90c9002e67efe6eff881dff4b73ed98fad3